### ISSUE TITLE
license issue

### ISSUE THREAD

**Issue by joudinet** [processone/p1_utils#1]
Your repository contains a copy of the Gnu Public License v2, but the source headers contain a reference to the Erlang Public License. Since the COPYING file has been added afterward, I guess you forgot to update the source headers, don't you?

**Comment by mremond** [comment]
No, p1_server and p1_fsm are forks coming from Erlang repository.

However, we are:
- Updating the license to Apache 2.0 to match Erlang one.
- As Erlang is now Apache 2.0, we should merge with Erlang R18 code.

**Comment by badlop** [comment]
> Updating the license to Apache 2.0 to match Erlang one.

Yes, this was done in 2ce13c8815f3aa76680d388f2a8cecced3ce8f42 and 18ffcf79e97311be6fe8b9899eac3a13b1b689ce

> As Erlang is now Apache 2.0, we should merge with Erlang R18 code.

Looking at the dates, maybe the files were forked from Erlang/OTP 17.3. At that time, merging with R18 was still possible. But nowadays, with R25 appearing seven years later, would that provide any benefit?
